src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -139,14 +145,17 @@ export class DotEditLayoutComponent implements OnInit {
      * @memberof DotEditLayoutComponent
      */
     setEditLayoutMode(): void {
+        this.initialFormValue.get('title').setValue(null);
```

Comment:
I completely miss this one in the past but... why are we saving in the ```initialFormValue``` the whole FormGroup instead of just the value? this seems like a waste of resources to me, can you do that change?